requirements/requirements.in

```diff
@@ -9,3 +9,4 @@ django-crum
 django-split-settings
 inflection
 sqlparse>=0.5.2 # https://github.com/ansible/django-ansible-base/security/dependabot/9
+dynaconf<4.0.0
```

Comment:
This is a requirement for all apps... because all apps have to use the DAB settings system?

Reply:
Yes, we expect all apps to load settings in the same standard, acceptance criteria is that DYNACONF is added as a requirement